flocker/provision/_install.py

```diff
@@ -519,14 +545,18 @@ def task_package_install(package_name, distribution,
     :return: a sequence of commands to run on the distribution
     """
     if package_source.branch:
-        # A development branch has been selected - add its Buildbot repo
```

Comment:
This function mixes in distro-agnostic code and distro-specific code. Would prefer if it was split out.